static/json/datadog-agent-ecs1-eu.json

```diff
@@ -33,17 +38,25 @@
           "name": "DD_API_KEY",
           "value": "<YOUR_API_KEY>"
         },
+        {
+          "name": "DD_SITE",
+          "value": "datadoghq.eu"
+        },
+        {
+          "name": "DD_LOGS_CONFIG_LOGS_DD_URL",
+          "value": "agent-intake.logs.datadoghq.eu:443"
```

Comment:
I'm not seeing the us version anymore. Am I missing it somewhere else?

Reply:
Didn't 6.8 make this easier with a single SITE variable? or is that 6.9? Can we document based on that?  Theres far too many options here.

Reply:
The dd_site variable works with 6.8 but doesn't support the log endpoint yet. This support will come with 6.9. Hence the need to have it set up independently for now.